File: OneDrive/Documentos/Store_Herencia/Store_Herencia/src/store/herencia/Person.java
package store.herencia;


public class Person {
    public String name;
    public String lastName;
    public String address;

    public Person(String name, String lastName, String address) {
        this.name = name;
        this.lastName = lastName;
        this.address = address;
    }

    public String showInfo() {
        return String.format("Nombre completo: %s %s", name, lastName);
    }

    public String getName() {
        return name;
    }

    public String getLastName() {
        return lastName;
    }

    public String getAddress() {
        return address;
    }
}

File: PycharmProjects/Store_Herencia/py/Person.py
class Person:
    def __init__(self, name, last_name, address):
        self.name = name
        self.last_name = last_name
        self.address = address

    def show_info(self):
        return f"Nombre completo: {self.name} {self.last_name}"

    def get_name(self):
        return self.name

    def get_last_name(self):
        return self.last_name

    def get_address(self):
        return self.address
